runtime/doc/options.txt

```diff
@@ -5918,6 +5918,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When on, splitting a window will put the new window right of the
 	current one. |:vsplit|
 
+			*'splitscroll'* *'spsc'* *'nosplitscroll'* *'nospsc'*
+'splitscroll' 'spsc'	number	(default 0)
+			global
+	The value of this option influences the scroll behavior when opening
```

Comment:
it doesn't "influence", it decides. Also don't need verbose preamble like "The value of this option...".

```suggestion
	Decides the scroll behavior when opening
```

Reply:
I would tend to agree but I tried to mimic the wording of other options:
https://github.com/neovim/neovim/blob/b78e4139b2914ed0e35f7643461561683c8051b7/runtime/doc/options.txt#L3657-L3664